src/entity/Post.php

```diff
@@ -0,0 +1,199 @@
+<?php
+
+namespace Projet5\entity;
+
+/**
+ * Post
+ */
+class Post
+{
+    private $id;
+    private $title;
+    private $last_date_change;
+    private $chapo;
+    private $contents;
+    private $publish;
+    private $id_bpf_users;
+
+    /**
+     * Method getId
+     * 
+     * Get value of idPost
+     *
+     * @return void
+     */
+    public function getId()
+    {
+        return $this->id;
+    }
+
+    /**
+     * Method setId
+     * 
+     * Set value of IdPost
+     *
+     * @param int $id
+     *
+     * @return void
+     */
+    public function setId($id)
+    {
+        $this->id = $id;
```

Comment:
```suggestion
        $this->id = $id;
        return $this;
```

idem pour tous les setters de ta classe

Reply:
return $this;    

Ajouté à tous les setters